cmd/dep/vndr_importer_test.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"log"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/golang/dep"
+	"github.com/golang/dep/internal/gps"
+	"github.com/golang/dep/internal/test"
+	"github.com/pkg/errors"
+)
+
+func TestVndrConfig_Import(t *testing.T) {
```

Comment:
@spenczar  Oops! I missed this comment until just now. Want to take a look at #992? I think we both approached it about the same.